package.json

```diff
@@ -202,9 +202,9 @@
     "@storybook/addon-essentials": "^7.0.0",
     "@storybook/addon-react-native-web": "0.0.19--canary.37.cb55428.0",
     "@storybook/addons": "^6.5.9",
-    "@storybook/builder-webpack5": "^6.5.10",
+    "@storybook/builder-webpack5": "^7.0.0",
     "@storybook/manager-webpack5": "^6.5.10",
-    "@storybook/react": "^6.5.9",
+    "@storybook/react": "^7.0.0",
```

Comment:
I was trying to update this in a different PR and this seems to break storybook, just so you are aware